build.sh

```diff
@@ -1,12 +1,17 @@
 #!/usr/bin/env bash
 
-set -feuo pipefail
+set -o errexit -o pipefail -o noglob -o noclobber -o nounset
 IFS=$'\n\t'
 
 readonly IMAGES=(
   alpine-nix
+  alpine-nix-rails-nginx
+  alpine-nix-rails-nginx/example
 )
 
+rm -rf alpine-nix-rails-nginx/example
+(cd alpine-nix-rails-nginx && rails new example --template example-rails-template.rb --skip-bundle)
```

Comment:
The main reason I include this is to give something to test with *before* the mrh-spree Dockerfile is created. Once it's created then this can be removed since we'll have a real-live system that we can verify the behaviour with.

> I wounder if we want to run the `rails new` from inside a booted container instance to make sure all version requirements apply.

I'd rather not go to the trouble of creating another docker container that is going to be thrown away.

What if I document that the `rails` gem is required in the `README`?